src/sentry/incidents/logic.py

```diff
@@ -387,6 +387,12 @@ def calculate_incident_time_range(incident, start=None, end=None, windowed_stats
 
             start = end - timedelta(seconds=time_window * WINDOWED_STATS_DATA_POINTS)
 
+    retention = quotas.get_event_retention(organization=incident.organization) or 90
+    if start < datetime.utcnow() - timedelta(days=retention):
+        start = datetime.utcnow() - timedelta(days=retention)
+        if end < start:
+            end = start
```

Comment:
```
start = max(start, datetime.utcnow() - timedelta(days=retention))
end = max(start, end)
```

Reply:
Oh, looks like we need to set the `tzinfo` on this so that we can compare properly, since it's causing CI issues